WordPress/Classes/ViewRelated/WhatsNew/WPWhatsNewView.m

```diff
@@ -0,0 +1,145 @@
+@property (nonatomic, weak, readwrite) IBOutlet NSLayoutConstraint *detailsHeightConstraint;
+@property (nonatomic, weak, readwrite) IBOutlet UIImageView* imageView;
+@property (nonatomic, weak, readwrite) IBOutlet UITextView* title;
+@end
+
+@implementation WPWhatsNewView
+
+#pragma mark - Initializers
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder
+{    
+    self = [super initWithCoder:aDecoder];
+
+    if (self) {
+        self.layer.cornerRadius = WPWhatsNewCornerRadiusDefault;
+        self.layer.masksToBounds = YES;
+    }
+
+    return self;
+}
+
+#pragma mark - NSObject
+
+- (void)awakeFromNib
+{
+    NSAssert([_details isKindOfClass:[UITextView class]],
+             @"Details outlet not wired.");
+    NSAssert([_imageView isKindOfClass:[UIImageView class]],
+             @"ImageView outlet not wired.");
+    NSAssert([_title isKindOfClass:[UITextView class]],
+             @"Title outlet not wired.");
+    
+    self.details.scrollEnabled = NO;
+    [self setTranslatesAutoresizingMaskIntoConstraints:NO];
+    
+    UIFont *titleFont = [WPFontManager openSansBoldFontOfSize:18.0f];
+    UIFont *detailsFont = [WPFontManager openSansLightFontOfSize:15.0f];
+    
+    self.title.font = titleFont;
+    self.details.font = detailsFont;
+}
+
+#pragma mark - Showing & hiding
+
+- (void)hideAnimated:(BOOL)animated
+          completion:(WPWhatsNewAnimationCompleteBlock)completion
+{
+    [UIView animateWithDuration:WPAnimationDurationFast
+                          delay:0
+                        options:UIViewAnimationOptionCurveEaseOut
+                     animations:^{
+                         self.alpha = WPAlphaZero;
+                     } completion:completion];
+}
+
+- (void)showAnimated:(BOOL)animated
+          completion:(WPWhatsNewAnimationCompleteBlock)completion
+{
+    self.transform = CGAffineTransformScale(CGAffineTransformIdentity,
+                                            WPWhatsNewShowAnimationMagnificationScale,
+                                            WPWhatsNewShowAnimationMagnificationScale);
+    self.alpha = WPAlphaZero;
+    
+    [UIView animateWithDuration:WPAnimationDurationFaster
+                          delay:0
+                        options:UIViewAnimationOptionCurveEaseOut
+                     animations:^void()
+    {
+        self.transform = CGAffineTransformIdentity;
+        self.alpha = WPAlphaFull;
+    } completion:completion];
+}
+
+#pragma mark - IBActions
+
+/**
+ *  @brief      Action to dismiss the popup.
+ *
+ *  @param      sender      The outlet that called this action.
+ */
+- (IBAction)dismissPopup:(id)sender
+{
+    if (self.willDismissBlock) {
+        self.willDismissBlock();
+    }
+    
+    __weak __typeof(self) weakSelf = self;
+    
+    [self hideAnimated:YES
+            completion:^(BOOL finished)
+    {
+        __strong __typeof(weakSelf) strongSelf = weakSelf;
+        
+        [strongSelf removeFromSuperview];
```

Comment:
Looks like the ```WPBackgroundDimmerView``` is not getting removed:

![screen shot 2015-01-23 at 11 52 00 am](https://cloud.githubusercontent.com/assets/154014/5879669/bcb46c16-a2f6-11e4-8616-15d78930b386.png)

![screen shot 2015-01-23 at 11 52 23 am](https://cloud.githubusercontent.com/assets/154014/5879670/c1fd78e8-a2f6-11e4-8dd7-50e97413c7af.png)